Lu/src/main/java/Controlador/Controlador.java

```diff
@@ -0,0 +1,109 @@
+package Controlador;
+import Enums.*;
+import Excepciones.InstrumentoDuplicadoException;
+import Excepciones.InstrumentoNoEncontradoException;
+import Excepciones.OpcionInvalidaException;
+import Vista.*;
+import Modelo.*;
+
+import java.util.List;
+
+public class Controlador {
+    private final Modelo modelo;
+    private final Vista vista;
+
+    public Controlador(Modelo modelo, Vista vista) {
+        this.modelo = modelo;
+        this.vista = vista;
+    }
+
+    private void registrar () throws InstrumentoNoEncontradoException, InstrumentoDuplicadoException {
+        InstrumentoFinanciero instrumentoFinanciero;
+        int instrumentoFinancieroTipo = vista.getTipo();
+        String nombre = vista.getNombre();
+        double precio = vista.getPrecio();
```

Comment:
Si bien esta perfecto el tema de que la Vista es la que obtiene el input,tener centralizado esta llamada a la vista en un metodo puntual en el controller puede prevenir futuras refatorizaciones.

```suggestion
        int instrumentoFinancieroTipo = ObtenerTipo();
        String nombre = ObtenerNombre();
        double precio = ObtenerPrecio();
```